src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -3101,16 +3231,74 @@ inline void rpp_resize_nn_load_i8pln1(Rpp8s *srcRowPtrsForInterp, Rpp32s *loc, _
 
 inline void rpp_store12_u8pkd3_to_u8pln3(Rpp8u* dstPtrR, Rpp8u* dstPtrG, Rpp8u* dstPtrB, __m128i &p)
 {
-    _mm_storeu_si128((__m128i *)(dstPtrR), _mm_shuffle_epi8(p, xmm_char_maskR)); /* Shuffle and extract the R pixels*/
-    _mm_storeu_si128((__m128i *)(dstPtrG), _mm_shuffle_epi8(p, xmm_char_maskG)); /* Shuffle and extract the G pixels*/
-    _mm_storeu_si128((__m128i *)(dstPtrB), _mm_shuffle_epi8(p, xmm_char_maskB)); /* Shuffle and extract the B pixels*/
+    _mm_storeu_si32((__m128i *)(dstPtrR), _mm_shuffle_epi8(p, xmm_char_maskR)); /* Shuffle and extract the R pixels*/
```

Comment:
please don't add spacing from previous (this has to be done as separate PR). When reviewing, it shows as difference in code and it is hard to review. It will be best to omit those changes

Reply:
Hi Rajy,
Here we changed from _mm_storeu_si128 to _mm_storeu_si32, since we want to store only 4 values
This is not a space change